.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           node-version: ${{ matrix.node }}
           cache: yarn
       - run: yarn install --immutable
+      - run: ./scripts/install-dev-on-example.sh
```

Comment:
Now we need to install the library and all the dependencies on the Example to accurately use the linter with type information